ambry-mysql/src/main/java/com/github/ambry/repair/MysqlRepairRequestsDb.java

```diff
@@ -205,6 +220,65 @@ public List<RepairRequestRecord> getRepairRequests(long partitionId) throws SQLE
     }
   }
 
+  /**
+   * Select the records from one partition but exclude the record with this source name and port.
+   * @param partitionId partition id
+   * @param sourceHostName the host name of the source replica
+   * @param sourceHostPort the host port of the source replica
+   * @return the oldest {@link RepairRequestRecord}s.
+   */
+  @Override
+  public List<RepairRequestRecord> getRepairRequestsExcludingHost(long partitionId, String sourceHostName,
+      int sourceHostPort) throws SQLException {
+    // private static final String GET_QUERY_EXCLUDE_SOURCE_REPLICA = String.format(""
+    //    + "SELECT %s, %s, %s, %s, %s, %s, %s "
+    //    + "FROM %s "
+    //    + "WHERE %s = ? and (%s != ? or %s != ?) "
+    //    + "ORDER BY %s ASC "
+    //    + "LIMIT ?",
+    //    BLOB_ID, SOURCE_HOST_NAME, SOURCE_HOST_PORT, OPERATION_TYPE, OPERATION_TIME, LIFE_VERSION, EXPIRATION_TYPE,
+    //    REPAIR_REQUESTS_TABLE,
+    //    PARTITION_ID, SOURCE_HOST_NAME, SOURCE_HOST_PORT,
+    //    OPERATION_TIME);
```

Comment:
nit: i think these statements are redundant.

Reply:
Will remove it in metrics PR.